qml/html/cm/codemirror.js

```diff
@@ -42,12 +43,15 @@
   var mobile = ios || /Android|webOS|BlackBerry|Opera Mini|Opera Mobi|IEMobile/i.test(navigator.userAgent);
   var mac = ios || /Mac/.test(navigator.platform);
   var windows = /win/i.test(navigator.platform);
-
+  
   var presto_version = presto && navigator.userAgent.match(/Version\/(\d*\.\d*)/);
   if (presto_version) presto_version = Number(presto_version[1]);
   if (presto_version && presto_version >= 15) { presto = false; webkit = true; }
+  
+  var qtVersion = navigator.userAgent.match(/QtWebEngine\/(\d*\.\d*.\d*)/);
+  qtVersion = qtVersion.length > 0 ? qtVersion[1] : '';
   // Some browsers use the wrong event properties to signal cmd/ctrl on OS X
-  var flipCtrlCmd = mac && (qtwebkit || presto && (presto_version == null || presto_version < 12.11));
+  var flipCtrlCmd = mac && (qtVersion.indexOf('5.4') != -1) && (qtwebkit || presto && (presto_version == null || presto_version < 12.11));
```

Comment:
better change to `indexOf('5.4') === 0` otherwise it will fail when version 5.5.4 comes out